package.json

```diff
@@ -8,6 +8,7 @@
     "@craco/craco": "^5.6.4",
     "@project-serum/serum": "0.13.1",
     "@project-serum/sol-wallet-adapter": "^0.1.1",
+    "@ezdefi/ez-sol-wallet-adapter": "git://github.com/ezdefi/ez-sol-wallet-adapter.git#0.1.1",
```

Comment:
choose something flexible for the tag name so we can update it later: `sereum` is a good choice, later we can keep the tag along with the latest branch